src/chains/tezos/tezos/tests/helpers/getProvider.ts

```diff
@@ -0,0 +1,25 @@
+import { utils } from "@ganache/utils";
+import TezosProvider from "../../src/provider";
+import { TezosProviderOptions } from "@ganache/tezos-options";
+const { RequestCoordinator, Executor } = utils;
+
+type Writeable<T> = { -readonly [P in keyof T]: T[P] };
+
+const getProvider = async (
+  options: Writeable<TezosProviderOptions> = {
+    wallet: { totalAccounts: 3, defaultBalance: 100 }
+  }
+) => {
+  const requestCoordinator = new RequestCoordinator(1);
```

Comment:
Let's use "no limit" here (0):

```suggestion
  const requestCoordinator = new RequestCoordinator(0);
```